Tools-RenameDocumentType-Umbraco/Program.cs

```diff
@@ -0,0 +1,127 @@
+﻿using Microsoft.Extensions.Configuration;
+using System.Globalization;
+
+namespace Umbraco.Doctype.Renamer
+{
+    internal class Program
+    {
+        private const string BASE_DIRECTORY = "BaseDirectory";
+        private const string ORIGINAL_DOC_TYPE_NAME = "OriginalDocTypeName";
+        private const string ORIGINAL_DOC_TYPE_ALIAS = "OriginalDocTypeAlias";
+        private const string NEW_DOC_TYPE_NAME = "NewDocTypeName";
+        private const string USYNC_DIRECTORY = "uSyncDirectory";
+        private const string IGNORE_CASE = "IgnoreCase";
+        private const string CULTURE_INFO = "CultureInfo";
+
+        static void Main(string[] args)
+        {
+
+            var configuration = new ConfigurationBuilder()
+                .AddJsonFile($"appsettings.json");
+
+            var config = configuration.Build();
+
+            var baseDirectory = GetConfigValueOrUserInput(config, BASE_DIRECTORY);
+
+            var originalDocTypeName = GetConfigValueOrUserInput(config, ORIGINAL_DOC_TYPE_NAME);
+            var originalDocTypeAlias = config[ORIGINAL_DOC_TYPE_ALIAS];
+            var originalDocTypeFileName = originalDocTypeName.Replace(" ", string.Empty);
+
+            if (string.IsNullOrEmpty(originalDocTypeAlias))
+            {
+                originalDocTypeAlias = GenerateAlias(originalDocTypeName);
+            }
+            
+            if (bool.TryParse(config[IGNORE_CASE], out var ignoreCase) == false)
+            {
+                Console.WriteLine("No value for 'IgnoreCase' found in appsettings.json, defaulting to true");
+                ignoreCase = true;
+            }
+
+
+            var cultureInfo = new CultureInfo(config[CULTURE_INFO] ?? "en-GB");
+
+
+            var newDocTypeName = GetConfigValueOrUserInput(config, NEW_DOC_TYPE_NAME);
+            var newDocTypeAlias = GenerateAlias(newDocTypeName);
+            var newDocTypeFileNamePrefix = GenerateFileNamePrefix(newDocTypeName);
+
+            var uSyncDirectory = GetConfigValueOrUserInput(config, USYNC_DIRECTORY);
+
+            var files = Directory.GetFiles(baseDirectory, $"*{originalDocTypeFileName}*", SearchOption.AllDirectories);
+
+            Console.WriteLine($"{files.Length} found");
+
+            foreach (var file in files)
+            {
+                Console.WriteLine($"{file} will be updated");
```

Comment:
Rather than printing to console what we are doing, could it be better to print it to a txt log? the Console window may get cluttered in instances theres a very large amount of files to update

Reply:
There shouldn't be a large number of files being updated. I think the information from the console output and the diff generated by git should be sufficient in telling a user what has happened. 

If better/ greater logging is required, we can add that as a separate PR